src/components/CodeView/index.tsx

```diff
@@ -134,39 +132,50 @@ export class CodeViewBase extends React.Component<Props> {
               <tbody className={styles.tableBody}>
                 {codeLines.map((code, i) => {
                   const line = i + 1;
-
-                  let rowProps: RowProps = {
-                    id: getCodeLineAnchorID(line),
-                    className: styles.line,
-                  };
-
-                  if (isLineSelected(rowProps.id, location)) {
-                    rowProps = {
-                      ...rowProps,
-                      className: makeClassName(
-                        rowProps.className,
-                        styles.selectedLine,
-                      ),
-                      ref: _scrollToSelectedLine,
-                    };
+                  const id = getCodeLineAnchorID(line);
+
+                  let className = styles.line;
+                  let shellRef;
+
+                  if (isLineSelected(id, location)) {
+                    className = makeClassName(
+                      styles.line,
+                      className,
```

Comment:
I'm confused. You set `className` to `styles.line` on line 137, and here you're using both `className` and `styles.line`, so that seems redundant. Can `className` be something other than `styles.line` at this point?

Reply:
Thanks. It was just a refactoring mistake.